.gitignore

```diff
@@ -7,4 +7,5 @@
 **/**/__pycache__
 .vscode/
 .pytest_cache
-src.egg-info
\ No newline at end of file
+src.egg-info
+**/migrations/
```

Comment:
@Dru94, do not add migrations in .gitignore
They need to be pushed for tracking